components/tokens/custom-express/custom-dark-vars.css

```diff
@@ -13,5 +13,7 @@ governing permissions and limitations under the License.
 
 .spectrum--express.spectrum--dark {
   /* Drop Zone background color rgb */
-  --spectrum-drop-zone-background-color-rgb: var( --spectrum-indigo-900-rgb); /* var(--spectrum-accent-color-900);*/
+  --spectrum-drop-zone-background-color-rgb: var(--spectrum-indigo-900-rgb); /* var(--spectrum-accent-color-900);*/
+
+  --spectrum-steplist-current-marker-color-key-focus: var(--spectrum-blue-700);
```

Comment:
Since these values do not change between Spectrum and Express, we should be able to define them once in the `custom-spectrum` files, and not have to define them here again. So, this line can be removed.